### PR TITLE
Adding Plume install steps for Yunohost

### PR DESCRIPTION
**Pull request by yalh76**

### CODE REVIEW

File: source/installation/with/yunohost.html.md [hunk 1]
```diff
@@ -2,8 +2,19 @@
 title: Installing with YunoHost
 ---
 
-This installation method is not yet documented. Your help would be welcome!
+You must have a Yunohost server.
```
Comment (igalic) [inline source/installation/with/yunohost.html.md]: "you need" might sound a bit better here.?

File: source/installation/with/yunohost.html.md [hunk 2]
```diff
@@ -2,8 +2,19 @@
...
-TODO: check this link is working
+If you don't have YunoHost, please see [here](https://yunohost.org/#/install) to know how to install and enjoy it.
 
-<a href="https://contribute.joinplu.me/#/documentation">Help writing this page</a>
\ No newline at end of file
+You can install it from the [Yunohost Apps](https://yunohost.org/#/apps_en), select All apps in the dropdown menu, Search for Plume, click on install and follow the steps
+
+or
+
+using from an SSH connexion to your Yunohost server:
```
Comment (igalic) [inline source/installation/with/yunohost.html.md]: please add a newline here for markdown processors which are not Github…

File: source/installation/with/yunohost.html.md [hunk 3]
```diff
@@ -2,8 +2,19 @@
...
\ No newline at end of file
+You can install it from the [Yunohost Apps](https://yunohost.org/#/apps_en), select All apps in the dropdown menu, Search for Plume, click on install and follow the steps
+
+or
+
+using from an SSH connexion to your Yunohost server: 
+```bash 
+sudo yunohost app install plume
+
+or
```
Comment (igalic) [inline source/installation/with/yunohost.html.md]: i would make this **two** `bash` sections, and leave the "or" out of it.

Comment (igalic) [reply source/installation/with/yunohost.html.md]: perhaps briefly explain what the difference is between the two methods:

"or, if you prefer to go bleeding edge,"

**Review by igalic — CHANGES_REQUESTED** [review]
👀

File: source/installation/with/yunohost.html.md (hunk 1 above)
Comment (yalh76) [reply source/installation/with/yunohost.html.md]: Yes without problem, made the change, my english is not subtle enough ^^

File: source/installation/with/yunohost.html.md (hunk 3 above)
Comment (yalh76) [reply source/installation/with/yunohost.html.md]: I removed the first way to install using SSH as the prerequisite is to add the community app list in Yunohost. But in a few weeks, community and official lists will be merged in a single list. So I let the universal way to install it.

File: source/installation/with/yunohost.html.md [hunk 4]
```diff
@@ -2,8 +2,16 @@
...
-TODO: check this link is working
+If you don't have YunoHost, please see [here](https://yunohost.org/#/install) to know how to install and enjoy it.
 
-<a href="https://contribute.joinplu.me/#/documentation">Help writing this page</a>
\ No newline at end of file
+You can install it from the [Yunohost Apps](https://yunohost.org/#/apps_en), select All apps in the dropdown menu, Search for Plume, click on install and follow the steps.
+
+or
+
+Using from an SSH connexion to your Yunohost server:
```
Comment (igalic) [inline source/installation/with/yunohost.html.md]: many people might be very confused with the rather rare spelling of "connexion"

please use "connection"

**Review by igalic — CHANGES_REQUESTED** [review]
spelling

File: source/installation/with/yunohost.html.md (hunk 4 above)
Comment (yalh76) [reply source/installation/with/yunohost.html.md]: My bad, usual french mistake

**Review by igalic — APPROVED** [review]
👍

**Comment by igalic** [comment]
thanks @yalh76

**Pull request merged**